src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -265,7 +266,15 @@ export function getExpandedResults(
         field = 'id';
       }
 
-      if (!field) {
+      // if at least one of the parameters to the function is an available column,
+      // then we should proceed to replace it with the column, however, for functions
+      // like apdex that takes a number as its parameter we should delete it
+      const {parameters = []} = AGGREGATIONS[exploded.function[0]] ?? {};
+      if (
+        !field ||
+        (!isEmpty(parameters) &&
```

Comment:
Would `parameters.length > 0` work here? It saves you an import and some boilerplate in the final code.